src/documents/css/idg.css.styl

```diff
@@ -355,11 +355,44 @@ setActiveCategory(inputColor) {
 
 /* Styles specific to small screens. Using Foundation 5 value for small screen width */
 @media only screen and (max-width: 40em) {
-    .docs-template-articleBody h1 {
-        font-size: 1.7rem;
+
+    * {
+        word-wrap: break-word;
```

Comment:
Do we really want text that looks like this?

![screen shot 2016-08-03 at 12 44 48 pm](https://cloud.githubusercontent.com/assets/556024/17373870/1e1e599c-5978-11e6-8cf3-2c555ad24b11.png)